pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1830,17 +1830,25 @@ func getDefaultNetworkTags(ctx context.Context, client client.Client, t *tag) ([
 	if err != nil {
 		return nil, errorUtil.Wrap(err, "failed to build default tags")
 	}
-	if t != nil {
-		return []*tag{
-			t,
-			croTag,
-			buildManagedTag(),
-		}, nil
-	}
-	return []*tag{
+	tags := []*tag{
 		croTag,
 		buildManagedTag(),
-	}, nil
+	}
+	if t != nil {
```

Comment:
NP, rename this to something a bit more meaningful, I know this is done all over CRO and some people like to super abbreviate variable names in GO, but it tends to not be the most readable code in the long run. So good habit to update stuff like this when you can.

Reply:
Agreed - updated 2 occurrences.